### ISSUE TITLE
Mesos for the unconvinced

### ISSUE THREAD

**Issue by roadan** [selagroup/sdp-june2016-cfp#77]
Distributed systems are everywhere. It doesn’t matter if you are running Big Data pipelines, NoSQL databases or large-scale web applications; you are building a distributed system.

In such a landscape, filled with cloud services, automaton frameworks and cluster managers, Apache Mesos is rapidly becoming the leading infrastructure for building and running distributed systems. Mesos is a cluster management system and framework that allow applications to schedule tasks over commodity hardware clusters in an isolated and safe manner. These applications can be applications written specifically for Mesos, or existing ones running on top of frameworks such as marathon, which manages Docker containers in a Mesos cluster. 

In this session, we will take a look of the unique design and capabilities of Mesos, and see how it can help running existing applications, and building new distributed system.

**Comment by goldshtn** [comment]
Just to make sure: this is a breakout session?

**Comment by roadan** [comment]
Yes it is, I've got one workshop, one session alone and one with Eyal (flnk)
